Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -26,6 +26,9 @@ extension MongoCollection {
         options: BulkWriteOptions? = nil,
         session: ClientSession? = nil
     ) -> EventLoopFuture<BulkWriteResult?> {
+        guard !requests.isEmpty else {
```

Comment:
following the "fail before the thread pool if possible" logic, I moved this back into `bulkWrite`